fbuapp/View Controllers/HomeViewController.m

```diff
@@ -18,27 +17,48 @@
 #import "PostTableView.h"
 #import "Masonry.h"
 #import "Favorite.h"
+#import "FilterViewController.h"
 
-@interface HomeViewController () <PostCellDelegate, TableViewDelegate>
+@interface HomeViewController () <PostCellDelegate, PostTableViewDelegate, FilterDelegate, CategoryHeaderViewDelegate>
+
+@property (strong, nonatomic) PFQuery *postQuery;
+@property PostTableView * feed;
 
 @end
 
 @implementation HomeViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    
-    PostTableView *feed = [[PostTableView alloc] initWithUserId:@"myuserid"]; //[PFUser currentUser].username
-    feed.tableViewDelegate = self;
-    [self.view addSubview:feed];
-    
-    [feed mas_makeConstraints:^(MASConstraintMaker *make) {
+    self.postQuery = [Post query];
+    self.feed = [[PostTableView alloc] initWithUserId:@"myuserid"]; //[PFUser currentUser].username
```

Comment:
What is this comment for?

Reply:
Once the current user is functioning [PFUser currentUser].username will replace @"myuserid"